packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -451,7 +506,11 @@ const VirtualListBaseFactory = (type) => {
 
 			primary.gridSize = primary.itemSize + spacing;
 			secondary.gridSize = secondary.itemSize + spacing;
-			thresholdBase = primary.gridSize * 2;
+			if (this.props.type === 'NewVirtualList') {
+				thresholdBase = primary.gridSize * Math.ceil(overhang / 2);
```

Comment:
I believe this line is the right code as we intended originally. I want this for other types of lists, too.

Reply:
`Math.ceil(overhang / 2)` is equal to `2` if this.props.type !== 'NewVirtualList'.
So let's remove `if`.